src/kem/newhope/optimized/rng.h

```diff
@@ -0,0 +1,49 @@
+//
+//  rng.h
```

Comment:
Is this file required for the build?  I think it is a left-over from the NIST submission.

Reply:
@smashra; I'll let you update this PR. Visual Studio doesn't refer to this file, so the Windows build should be ok.